src/absil/ildiag.fs

```diff
@@ -23,3 +23,21 @@ let dprintfn (fmt: Format<_,_,_,_>) =
     Printf.kfprintf dflushn (match !diagnosticsLog with None -> System.IO.TextWriter.Null | Some d -> d) fmt
 
 let setDiagnosticsChannel s = diagnosticsLog := s
+
+//---------------------------------------------------------------------
+// Library: ReportTime
+//---------------------------------------------------------------------
+
+let reportTime =
```

Comment:
This should probably go in ``illib.fs``.  The file ``ildiag.fs`` is essentially on-the-way-out and I've been slowly getting rid of it.